quarterback-packages/api/src/PromClientRegistryConfig.ts

```diff
@@ -59,24 +59,24 @@ function register(
   }
 }
 
-export async function configurePromClientRegistry(): Promise<void> {
-  const appVersion: SemVer | null = parse(packageJson.version, {})
-  if (appVersion) {
-    register(appMetricMap, appVersion)
-  }
-  for (const metric of metricsMap) {
-    const semver = await getVersion(metric.packageName)
-    if (semver) {
-      register(metric, semver)
-    }
-  }
-}
+// export async function configurePromClientRegistry(): Promise<void> {
+//   const appVersion: SemVer | null = parse('0.0.0', {})
+//   if (appVersion) {
+//     register(appMetricMap, appVersion)
+//   }
+//   for (const metric of metricsMap) {
+//     const semver = await getVersion(metric.packageName)
+//     if (semver) {
+//       register(metric, semver)
+//     }
+//   }
+// }
 
-async function getVersion(packageName: string): Promise<SemVer | null> {
-  const packageFile = await fs.readFile(
-    `./quarterback-packages/api/node_modules/@manuscripts/${packageName}/package.json`,
```

Comment:
@mbetamony this is related to making @manuscripts/transform work in the api. I just commented it because it didn't work but we actually shouldn't be using such relative paths. Path to node_modules has to be resolved and not hardcoded.